src/components/Root.tsx

```diff
@@ -1,17 +1,20 @@
+import '@ag-grid-community/styles/ag-grid.min.css';
+import '@ag-grid-community/styles/ag-theme-material.min.css';
+import '@uppy/core/dist/style.css';
+// hack to have the text editor styles available without having to bundle them with graasp/ui
+import 'katex/dist/katex.min.css';
+import 'react-quill/dist/quill.snow.css';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
Was it better than building it in ui? Because now we have to do it for all platforms? or is it only necessary on edit?

Reply:
Building it in UI made a big CSS bundle with all of them, using them here allows to be a bit ore selective... 
I also was not able to make vite detect that there was the styles.css file available in the ui package ... it was failing with an error from the dependencies package ... apparently it is a bug on vite's side, so I avoided it completely this way. 

Also you only need to import the quill styles if you plan on displaying text from quill, same for katex.